### ISSUE TITLE
Error in upgrading plugin

### ISSUE THREAD

**Issue by mguarrasi** [aiidateam/aiida-core#3208]
Dear All,

I upgraded my aiida-quantumespresso plugin and after this also the aiidacore.
When I tried to to run verdi I obtained the following error:

verdi
Traceback (most recent call last):
  File "/home/ubuntu/aiidapy/bin/verdi", line 6, in <module>
    from aiida.cmdline.commands.cmd_verdi import verdi
  File "/home/ubuntu/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/commands/__init__.py", line 21, in <module>
    from aiida.cmdline.commands import (cmd_calcjob, cmd_code, cmd_comment, cmd_completioncommand, cmd_computer, cmd_config,
  File "/home/ubuntu/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/commands/cmd_calcjob.py", line 31, in <module>
    @arguments.CALCULATION('calcjob', type=CalculationParamType(sub_classes=('aiida.node:process.calculation.calcjob',)))
  File "/home/ubuntu/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/params/types/identifier.py", line 67, in __init__
    raise ValueError('{} is not a valid entry point string: {}'.format(entry_point_string, exception))
ValueError: aiida.node:process.calculation.calcjob is not a valid entry point string: Entry point 'process.calculation.calcjob' not found in group 'aiida.node'

This seems to be related to issue #632, but I don't know where I must add the row reentry_scan=['aiida'] to fix the issue. 
In other words where I can find the setup.py file of aiida-core.

Thanks.

Cheers,
M.

**Comment by sphuber** [comment]
Please try to run `reentry scan`. This should fix it

**Comment by mguarrasi** [comment]
Thanks Sebastian. Now it runs! 

Cheers,
M.
